nbbuild/build.xml

```diff
@@ -138,6 +138,7 @@
     <property file="${nb_all}/nbbuild/build/netbeansrelease.properties"/>
     <!-- javadoc content filtering -->
     <setjavadoc branch="${metabuild.branch}"/>
+    <antcall target="-load-module-flags"/>
```

Comment:
does this work? I thought antcall would run in its own context and the property would be empty after it. (I am a bit rusty with ant)

Reply:
it works but the -load module append the file I guess that replacement reread the file